engine/src/main/resources/assets/ui/menu/settingsMenuScreen.ui

```diff
@@ -60,10 +60,26 @@
                         "id": "input",
                         "text": "${engine:menu#input-settings}"
                     },
+                    {
+                        "type": "UIButton",
+                        "id": "test",
+                        "text": "autoconfig"
+                    },
+                    {
+                        "type": "UIButton",
+                        "id": "test2",
+                        "text": "math"
+                    },
+                    {
+                        "type": "UIButton",
+                        "id": "test3",
+                        "text": "buildin"
+                    },
```

Comment:
As above; I don't think we should make these test screens visible in the main game UI. If we need to view them, we can always use the console command to load them.

Reply:
Debug screen for presentation.
should will remove before merge